### ISSUE TITLE
Fira Sans spacing issues

### ISSUE THREAD

**Issue by madig** [system76/pop-gtk-theme#40]
I just came across the Pop blog post and wanted to chime in: Fira Sans, as rendered on a Linux desktop, is suffering from spacing issues that may not be the fault of the typeface itself, but rather the graphics library that draws it. Compare the following screenshots from Linux...
![firasans-linux](https://user-images.githubusercontent.com/380829/27502300-5e157346-5873-11e7-864d-56aaaef18ce3.png)
and macOS: 
![firasans-macos](https://user-images.githubusercontent.com/380829/27502314-6e37d2fa-5873-11e7-87b2-8af802a00485.png)
Note how much more even the rendering is on macOS. Also look for words like "mode" and "change" and compare the spacing between glyphs. I guess that subpixel positioning of glyphs is to blame, which cairo (-> Gtk) doesn't do, Qt 5.x does it to a certain extend at least.

The bottom line is that this hurts your consistency goal. I have no easy solution to this. A medium solution would be to use the (trial version of the) Glyphs app on a Mac and use the open-source HT Letterspacer plugin to respace the masters and generate the fonts that you need (and rename them!). Which is not pretty but gets the job done.

**Comment by isantop** [comment]
Which browsers were these rendered in? I'll agree that Firefox (somewhat oddly, considering the origins of Fira) doesn't do a great job of rendering the font, but in the rest of the GNOME desktop this seems to be much more consistent. For example, both of those words as rendered within GTK:

![screenshot from 2017-06-26 11-02-39](https://user-images.githubusercontent.com/5883565/27551040-19efdb1e-5a5f-11e7-86a9-06b00f732efa.png)

Not perfect, but better.

Fira is a part of the System76 brand, and thus should belong in Pop. Manually editing the font to make it better render is a very large task for very small benefit, given the improved rendering in GTK versus Firefox.
